connector-email/src/main/java/com/redhat/cloud/notifications/connector/email/EmailRouteBuilder.java

```diff
@@ -74,23 +75,27 @@ public void configureRoutes() {
                 .to(direct(ENTRYPOINT));
         }
 
-        /*
-         * Prepares the payload accepted by BOP and sends the request to
-         * the service.
-         */
-        final HttpEndpointBuilder bopEndpointV1 = setUpBOPEndpointV1();
-
         from(seda(ENGINE_TO_CONNECTOR))
             .routeId(emailConnectorConfig.getConnectorName())
-            .process(recipientsResolverRequestPreparer)
-            .to(RECIPIENTS_RESOLVER_RESPONSE_TIME_METRIC + TIMER_ACTION_START)
-                .to(setupRecipientResolverEndpoint())
-            .to(RECIPIENTS_RESOLVER_RESPONSE_TIME_METRIC + TIMER_ACTION_START)
-            .process(recipientsResolverResponseProcessor)
-            .choice().when(shouldSkipEmail())
-                .log(INFO, getClass().getName(), "Skipped Email notification because the recipients list was empty [orgId=${exchangeProperty." + ORG_ID + "}, historyId=${exchangeProperty." + ID + "}]")
-            .otherwise()
-                .to(direct(Routes.SPLIT_AND_SEND))
+            .choice()
+                .when(shouldUseSimplifiedEmailManagement())
+                    .process(emailManagementProcessor)
+                .endChoice()
```

Comment:
Each `when` and `otherwise` cases have to be closed explicitly when a sub `choice` is involve.